pkg/webhook/add_customresourcedefinition.go

```diff
@@ -0,0 +1,8 @@
+package webhook
```

Comment:
Hi Arvind! If I understand it correctly, should this file be named `add_resourcehandler.go` instead?